CRM/Core/Session.php

```diff
@@ -547,6 +547,21 @@ public static function getLoggedInContactID() {
   }
 
   /**
+   * Get display name of the logged in user.
+   *
+   * @return string
+   *
+   * @throws CiviCRM_API3_Exception
+   */
+  public function getLoggedInContactDisplayName() {
```

Comment:
changing the method name to **getLoggedInUserDisplayName()**   will make it more expressive  , or maybe just **getUserDisplayName()** should be enough.